src/TensorCore.jl

```diff
@@ -4,6 +4,8 @@ using LinearAlgebra
 
 export ⊙, hadamard, hadamard!
 export ⊗, tensor, tensor!
+export ⊡, boxdot, boxdot!
+export @pirate
```

Comment:
What's this?

Reply:
It was a macro which made `ones(1,2,3)'` reverse dimensions. But I deleted it as being too weird.

The function `_adjoint(A)` is still there, although more care may be needed in order for `⊡` to play reasonably well with `PermutedDimsArray{(3,2,1)}`.

Reply:
I've removed this export. And decided that making PermutedDimsArray work nicely is going to take more thought.